<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>EFETUAR SAQUE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn20">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R$ 20</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn50">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R$ 50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn100">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R$ 100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn150">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>R$ 150</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtValorSaque">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIRMAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>LIMPAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>VALOR DO SAQUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R$</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Informar apenas um valor inteiro</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSaldo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>SEU SALDO: R$</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SAQUE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSaque">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
